bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterUtilities.java

```diff
@@ -73,6 +70,66 @@ public static BigtableClusterUtilities forAllInstances(String projectId)
   }
 
   /**
+   * @return The instance id associated with the given project, zone and cluster.
+   */
+  public static String lookupInstanceId(String projectId, String clusterId, String zoneId)
+    throws IOException {
+    BigtableClusterUtilities utils;
+    try {
+      utils = BigtableClusterUtilities.forAllInstances(projectId);
+    } catch (GeneralSecurityException e) {
+      throw new RuntimeException("Could not initialize BigtableClusterUtilities", e);
+    }
+
+    try {
+      Cluster cluster = utils.getCluster(clusterId, zoneId);
```

Comment:
Maybe add comment like this:
Since we don't have instance ID, we first list all clusters in the project and try to match by cluster by cluster ID. There must be one and only one matching cluster.